### PR TITLE
Remove unnecessary checklist items

### PR DESCRIPTION
**Pull request by SUPERCILEX**
@andrewda How does the release process work, is it automatic? `#4` is unnecessary since that's what the codeowners file is for and `#3` is meh since we can just squash. I disagree that the burden to make nice commits should be on the contributor, it should be on the maintainer. What do you think?

### CODE REVIEW

**Comment by SUPERCILEX** [comment]
Also, @sertbot doesn't work on https://github.com/SouthEugeneRoboticsTeam/sertbot? 😁

**Comment by andrewda** [comment]
I'll get it working on this repo, it's just all kinda set up for the actual robot right now. I'll need to make a `robot` setting vs. `library` or repository.

File: lib/defaults.js [hunk 1]
```diff
@@ -28,9 +28,7 @@ module.exports = {
   checklists: {
     change: [
       'Test code changes on the robot (if possible).',
-      'Verify that significant changes have tests and pass successfully.',
-      'Rebase commits into one if neccessary. Verify that it is appropriatly titled.',
-      'Request code reviews from other team members working on the same area of the robot (a review from @SouthEugeneRoboticsTeam/reviewers is required).'
+      'Verify that significant changes have tests and pass successfully.'
```
Comment (andrewda) [inline lib/defaults.js]: Maybe add: "Ensure this pull request is appropriately titled." or something like that?

Comment (SUPERCILEX) [reply lib/defaults.js]: Oh, I see what you mean. Yeah, totally!

**Review by andrewda — APPROVED** [review]

**Pull request merged**